src/app-layer-detect-proto.c

```diff
@@ -1960,7 +1960,9 @@ int AppLayerProtoDetectConfProtoDetectionEnabled(const char *ipproto,
 
         node = ConfGetNode(param);
         if (node == NULL) {
-            SCLogDebug("Entry for %s not found.", param);
+            SCLogWarning(SC_ERR_CONF_YAML_ERROR,
+                    "Proto entry not found in suricata.yaml. Enabling anyway."
+                    "Behavior to be discontinued in Suricata 7. Update configuration.");
```

Comment:
@victorjulien do you think this is appropriate? Should the message or log level or both be changed?

Reply:
How about:

"Config for protocol %s not found, so enabling by default. This behavior will change in Suricata 7, so please update your config. See ticket #4744 for more details."

Where a more thorough explanation of the issue and how to suppress the warning should be in the ticket.

Reply:
OK. Thank you!